tests/e2e/utils/utils.js

```diff
@@ -167,8 +167,15 @@ async function makeRequest(page, endpoint, apiVersion = "v0") {
     const url = host + apiVersion + endpoint;
     console.log("makeRequest", url);
     const resp = await fetch(url);
-    const jsonResp = await resp.json();
-    return jsonResp["data"];
+
+    try {
+      const jsonResp = await resp.json();
+      return jsonResp["data"];
+    }
+    catch(error) {
+      console.log("-- No JSON in response --");
```

Comment:
Can you log the text of the response? Ideally the headers and the body. This way we can see whether a) it really comes from "trafik" and b) the ``S`` comes from ``Service Unavailable``

Reply:
The critical method that calls it is already logging the response, but I guess it doesn't harm if we also do it here.